app/src/main/java/org/mifos/mobile/ui/about/AboutUsFragment.kt

```diff
@@ -0,0 +1,87 @@
+package org.mifos.mobile.ui.about
+
+import android.app.Activity
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.material3.Scaffold
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.platform.ViewCompositionStrategy
+import androidx.fragment.app.viewModels
+import com.google.android.gms.oss.licenses.OssLicensesMenuActivity
+import dagger.hilt.android.AndroidEntryPoint
+import org.mifos.mobile.core.ui.theme.MifosMobileTheme
+import org.mifos.mobile.ui.activities.PrivacyPolicyActivity
+import org.mifos.mobile.ui.enums.AboutUsListItemId
+import org.mifos.mobile.ui.fragments.base.BaseFragment
+import org.mifos.mobile.utils.Constants.LICENSE_LINK
+import org.mifos.mobile.utils.Constants.SOURCE_CODE_LINK
+import org.mifos.mobile.utils.Constants.WEBSITE_LINK
+
+/*
+~This project is licensed under the open source MPL V2.
+~See https://github.com/openMF/self-service-app/blob/master/LICENSE.md
+*/
+@AndroidEntryPoint
+class AboutUsFragment : BaseFragment() {
+
+    private val viewModel: AboutUsViewModel by viewModels()
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?,
+    ): View {
+        return ComposeView(requireContext()).apply {
+            setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
+            setContent {
+                MifosMobileTheme {
+                    Scaffold {
+                        AboutUsScreen()
```

Comment:
Pass the viewModel instance in `AboutScreen`, you should use the same instance of the view model in both fragment and the compose screen